app/react/V2/CustomHooks/useDnDContext.tsx

```diff
@@ -0,0 +1,53 @@
+/* eslint-disable max-statements */
```

Comment:
I don't believe this hook will ever be used outside of a drag and drop functionality. It would be ok in my opinion to move this inside the folder with the rest of the dnd components and functionality. That way you could import everything from `app/V2/Components/Layouts/DragAndDrop`.